sunpy/data/data_manager/manager.py

```diff
@@ -27,11 +27,13 @@ def __init__(self, cache):
         self._namespace = None
         self._skip_hash_check = False
         self._skip_file = {}
+        self._require_files = {}
 
-    def require(self, name, urls, sha_hash):
+    def require(self, name, urls, sha_hash, defer_download=False):
         """
         Decorator for informing the data manager about the requirement of
-        a file by a function.
+        a file by a function. Optionally defer downloading the file until it's
+        requested in `get`.
```

Comment:
I don't think this link will work.

Reply:
Could you please provide additional information regarding the link?

Reply:
I just mean that the interlink from sphinx won't work with just get. You will need to add the full path, even assuming that the get is even documented in the API docs.